modules/ztp-validating-the-generation-of-installation-crs.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-installation-crs_{context}"]
+
+= Validating the generation of installation CRs
+
+SiteConfig applies Installation custom resources (CRs) to the hub cluster in a namespace with the name matching the site name. To check the status, use the following command:
+
+[source,terminal]
+----
+$ oc get AgentClusterInstall -n <clusterName>
+----
+
+If no object is returned, use the following procedure to troubleshoot the ArgoCD pipeline flow from SiteConfig to installation CRs.
```

Comment:
from SiteConfig to the installation CRs. ? Should there be an article there?

Reply:
fixed